site/app/models/User.php

```diff
@@ -35,6 +35,8 @@
  * @method void setGroup(integer $group)
  * @method void setRegistrationType(string $type)
  * @method string getRegistrationSection()
+ * @method string setCourseSectionId()
+ * @method void setCourseSectionId(string $Id)
```

Comment:
Is the first one supposed to be `getCourseSectionId()`?

Reply:
Yes, you're right